cogs/player/jockey.py

```diff
@@ -632,3 +639,47 @@ async def update_now_playing(self):
                     self.guild.name,
                     exc
                 )
+
+    async def play_bump(self) -> bool:
+        """
+        Check and attempt to play a bump if it's been long enough.
+        :return: A boolean denoting whether a bump was played or not.
+        """
+
+        enabled = self._db.get_bumps_enabled(self.guild.id)
+        if not enabled:
+            self._logger.debug("Bumps are not enabled in this guild, returning...")
+            return False
+
+        interval = self._db.get_bump_interval(self.guild.id) * 60
+        last_bump = self._db.get_last_bump(self.guild.id)
+
+        if last_bump == 0:
+            self._logger.debug('Last bump is zero, initializing and skipping...')
+            self._db.set_last_bump(self.guild.id)
+            return False
+
+        if int(time()) - last_bump < interval:
```

Comment:
Correct me if my intuition is wrong, but does this mean Jockey will insert a bump in between a track that's just finished and the track following it, if it's been long enough since the last bump?

If yes, would you know what the effect is on manual queue manipulation - that is, does skipping to the previous track cause Blanco to play the bump instead? Or is the QueueManager completely untouched here?